migrations/Version202402231420400684_qtiItemPci.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\scripts\install\RegisterPciLikertScore;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202402231520001465_qtiItemPci extends AbstractMigration
```

Comment:
Should match file name? Though I don't think it matters.
`Version202402231520001465_qtiItemPci` --> `Version202402231420400684_qtiItemPci`

Reply:
It should. I will change it depending on the timeline of #397.